az_success.message=Əməliyyat uğurla tamamlandı
az_bad_request.message=Məlumatın doğruluğunu yoxlayın
az_error_no_such_push_notification.message=Məlumatın doğruluğunu yoxlayın
az_error_internal.message=Əməliyyat zamanı xəta baş verdi. Yenidən cəhd etməyiniz xahiş olunur.

# UserDto Messages
Size.cif=CIF uzunluğu 7 simvoldan ibarət olmalıdır.
NotNull.platform=Platform daxil etməlisiniz.

user.created.message=İstifadəçi uğurla yaradıldı.
user.updated.message=İstifadəçi uğurla yeniləndi.
user.topics.set.successfully=Topic-lər istifadəçiyə uöurla təyn edildi.

topic.deleted.success.message=Topic uöurla silindi.
topic.fetched.success.message=Topic-lər uğurla çəkildi.
topic.created.success.message=Topic uğurla yaradıldı.
topic.exists=Topic artıq mövcuddur.


